tutorials/model_editor.md

```diff
@@ -0,0 +1,76 @@
+\page model_editor Model Editor
+
+## Overview
+
+Simulation worlds and models in Gazebo are defined using [SDF](http://sdformat.org), an XML-based file format. While these files are
+human-readable, they can be challenging to manually parse or create. Gazebo
+has introduced graphical tools to ease the process, allowing users to
+construct worlds and models with greater ease.
+
+## Component inspector editor
+
+Graphical model editing capabilities are available through the `Component
+Inspector Editor` GUI plugin, rather than the more commonly used `Component
+Inspector`. This distinction is relevant during the development of editing
+features. To access the features discussed in this tutorial, ensure you load
+the `Component Inspector Editor` plugin.
+
+## Simulation state
+
+The ability to add and edit certain items via the GUI depends on whether the
+simulation is running or paused. This is crucial due to the interplay of
+physics with edits made through the GUI. For instance, major changes to
+physics, like adding or removing a link from a model, generally cannot occur
+while the simulation is running.
+
+While simulation is running, you can create new models and change the pose
+of a model. However, most other operations are not permitted in the running
+state. Graphical elements that are not available during runtime will be
+disabled.
+
+## Adding a model or light
+
+To insert a model or light into a world, use the `+` button on the `Entity Tree`.
+
+@image html files/model_editor/entity_tree_plus.png "Entity tree plus" width=60%
+
+Selecting the `+` button opens a menu listing different types of models and
+lights. Choose an item from the menu, move your mouse cursor in the 3D scene
+to place the new object, and left-click to confirm. The new object will
+appear in the `Entity Tree`, and you can start or continue modifying the
+simulation content.
+
+## Adding a link, joint, or light to a model
+
+To add entities to a model, simulation must be paused. Select a model in
+the Entity Tree, then click the + button on the Component Inspector Editor
+plugin. In the example below, we are adding an entity to the selected box
+entity.
+
+@image html files/model_editor/add_link.png "Add Link" width=60%
+
+A menu displaying different types of links, joints, and lights will appear.
+Select the desired entity type to add to the model. The new entity will
+appear as a child of the selected model in the `Entity Tree`, and the 3D scene
+will display the new entity. The GUI is aware of the entity's existence at
+this point, but the server is not. Upon pressing play or step, the GUI
```

Comment:
For some reason, after I add a model, if I add a link and/or joint, or a sensor, then hit Play, the model disappears altogether. Does that happen to you, or is this my local setup?

Steps to reproduce:
1. While paused, Entity Tree panel + button, add a Box to the scene. Hit Play. Box stays in scene.
2. Pause physics. Select box. Component Inspector Editor panel + button > Link > Box. A new box link is added.
3. Hit Play. The whole box model disappears, that is, both the new box and the original box links.
I tried editing the pose of the new box while pause, in case it's a collision problem.
I also tried adding a joint between the original link and the new link. The model still disappears.
Similar happens to sensors.

Reply:
Indeed, that doesn't seem to work. I'll close this PR.